plugins/mollie/config/locales/nl.yml

```diff
@@ -0,0 +1,44 @@
+nl:
+  activerecord:
+    attributes:
+      home:
+        credit_your_account: Geld storten
+  home:
+    index:
+      credit_your_account: Geld storten
+    ordergroup:
+      credit_your_account: Geld storten
+  payments:
+    mollie:
+      new:
+        title: Stort geld op je account
+        no_ordergroup: U dient lid te zijn van een huishouden
+      create:
+        invalid_amount: Ongeldig bedrag
+      form:
+        amount_pay: Te betalen bedrag
+        method: Hoe te betalen
+        submit: Betaal online
+        financial_transaction_type: Financiële-transactietype
+        fee: Selecteer de juiste transactiekosten
+      controller:
+        result:
+          notice: Er is €%{amount} bijgeschreven op uw account.
```

Comment:
Please use the standard way to format currencies, as the currency may not be €.
Ok, later I read that Mollie (currently) assumes EUR (is that still the case?), so maybe not important. But the english language does not have the currency, so good to bring in harmony.

Reply:
oh yes, thats now taken from the payment_currency field, just forgot to adapt the nl translation. 
The support for automatic fee calculation is currently only supported for EUR, because of api limitation